crates/sovereign-sdk/module-system/sov-modules-rollup-blueprint/src/lib.rs

```diff
@@ -60,8 +60,15 @@ pub trait RollupBlueprint: Sized + Send + Sync {
     /// Creates a new instance of the blueprint.
     fn new() -> Self;
 
-    /// Get code commitments by fork.
-    fn get_code_commitments_by_spec(&self) -> HashMap<SpecId, <Self::Vm as Zkvm>::CodeCommitment>;
+    /// Get batch prover code commitments by fork.
+    fn get_batch_prover_code_commitments_by_spec(
+        &self,
+    ) -> HashMap<SpecId, <Self::Vm as Zkvm>::CodeCommitment>;
+
+    /// Get light client prover code commitments by fork.
+    fn get_light_client_prover_code_commitments_by_spec(
+        &self,
+    ) -> HashMap<SpecId, <Self::Vm as Zkvm>::CodeCommitment>;
```

Comment:
I am not sure how light client forks are gonna work. so let's make this a single code commitment for now